features/encrypt/core/src/main/java/org/apache/shardingsphere/encrypt/merge/dql/EncryptMergedResult.java

```diff
@@ -66,7 +69,14 @@ public Object getValue(final int columnIndex, final Class<?> type) throws SQLExc
         EncryptColumn encryptColumn = encryptRule.getEncryptTable(originalTableName).getEncryptColumn(originalColumnName);
         String schemaName =
                 selectStatementContext.getTablesContext().getSchemaName().orElseGet(() -> new DatabaseTypeRegistry(selectStatementContext.getDatabaseType()).getDefaultSchemaName(database.getName()));
-        return encryptColumn.getCipher().decrypt(database.getName(), schemaName, originalTableName, originalColumnName, cipherValue);
+        try {
+            return encryptColumn.getCipher().decrypt(database.getName(), schemaName, originalTableName, originalColumnName, cipherValue);
+            // CHECKSTYLE:OFF
+        } catch (final Exception ex) {
```

Comment:
Can one or more specific exceptions be thrown?

Reply:
AES is found to throw a GeneralSecurityException, which is a checked exception and would be inconvenient to wrap as a runtime exception. Additionally, considering the custom algorithm, it is challenging to predict the exact exception type.